scss/README.md

```diff
@@ -0,0 +1,20 @@
+This directory is for SASS, using the structure outlined at
+http://thesassway.com/beginner/how-to-structure-a-sass-project
+
+How to compile SASS
+==
+
+See https://www.jetbrains.com/help/phpstorm/2016.3/transpiling-sass-less-and-scss-to-css.html for instructions on how to compile SASS to CSS in phpstorm
+
+To compile the SASS into a CSS file using Grunt, run the following command:
+```
+grunt sass
```

Comment:
There is a phing-buildfile that should contain a task/target to do this compilation. At least as an add-on